src/patchRequire.js

```diff
@@ -1,6 +1,7 @@
 import * as path from 'path';
 
-
+const unixify = process.platform === 'win32' ? require('unixify') : (p)=>p;
+const correctPath = process.platform === 'win32' ? (path) => unixify( path.replace(/^\\\\\?\\.:\\/,'\\')) : (p)=>p;
```

Comment:
I was seeing paths like  `\\?\c:\foo\bar`.

This trims the `\\?\` and unixifys the rest.